src/mbgl/programs/symbol_program.hpp

```diff
@@ -8,6 +8,7 @@
 #include <mbgl/util/geometry.hpp>
 #include <mbgl/util/size.hpp>
 #include <mbgl/style/layers/symbol_layer_properties.hpp>
+#include <mbgl/style/layers/symbol_layer_impl.hpp>
```

Comment:
This is needed here because it's where `style::{Icon,Text}PaintProperties` are declared.